analyticsclient/engagement_timeline.py

```diff
@@ -0,0 +1,29 @@
+import urllib
+from analyticsclient.base import PostableCourseIDsEndpoint
+from analyticsclient.constants import data_formats
+
+
+class EngagementTimeline(PostableCourseIDsEndpoint):
+    """Engagement Timeline."""
+
+    def __init__(self, client, username, course_id):
+        """
+        Initialize the EngagementTimeline client.
+
+        Arguments:
+
+            client (analyticsclient.client.Client): The client to use to access remote resources.
+            username (str): String identifying the user (e.g. jbradley)
+            course_id (str): String identifying the course (e.g. edX/DemoX/Demo_Course)
+
+        """
+        super(EngagementTimeline, self).__init__(client)
+
+        self.username = unicode(username)
+        self.course_id = unicode(course_id)
+
+    def get(self):
```

Comment:
I want to explicitly point out the name of this method - this design is different than what we do in other areas of `edx-analytics-data-api-client` ([like `Programs`, for example](https://github.com/edx/edx-analytics-data-api-client/blob/master/analyticsclient/programs.py#L10)).

I think `self.client.engagement_timeline(username, course_id).engagement_timeline()` is less readable than `self.client.engagement_timeline(username, course_id).get()` but happy to change this.

Reply:
Yeah that's fair. I'm a bit hesitant because of the inconsistency it creates with the rest of the endpoints, but I think it's better to make this one class more readable. @dsjen what do you think?

Reply:
Hmm, I have the same conundrum as @thallada. `get` is a fair way to go here from my perspective too!